flask_server.py

```diff
@@ -1,59 +1,68 @@
-from Instastories import scrape_from_web
-from flask import Flask, render_template, request, url_for, flash
+from Instastories import start_scrape
+from flask import Flask, render_template, request, url_for, Markup
 import os
 import json
 
 app = Flask(__name__)
 
 ################### UTIL FUNCTIONS ###################
 
-def setLogFileList():
+def set_log_file_list():
     log_line = []
-
     if not os.path.exists("run_history.log"):
         open("run_history.log", "w").close()
-
     with open("run_history.log", "r") as o:
         return [log_line for log_line in o.readlines()]
 
 
-def saveSettings(settings):
+def save_settings(settings):
     with open("settings.json", "w+") as settings_json:
         json.dump(settings, settings_json)
 
-def getSettings():
-    if os.path.exists("settings.json"):
-        with open("settings.json", "r") as settings_json:
-            settings = json.load(settings_json)
-        return settings
-    else: return {}
-
+def get_settings():
+    if not os.path.exists("settings.json"):
+        return {}
+    with open("settings.json", "r") as settings_json:
+        return json.load(settings_json)
+
+def render_base64_media(base64_media):
+    """
+    base64_media structure: 
+        0 - Data
+        1 - content type
+        2 - username
+    """
+    rendered_base64_media = []
+    for media in base64_media:
+        content_tag = "img" if "img" in media[1] else "video controls"
```

Comment:
media[1] =="img"  (It's not a list)
With the above suggestion media_type == "img"